### ISSUE TITLE
[Bug][CLI][4.6.0] Fails to work with Ledger

### ISSUE THREAD

**Issue by gregnazario** [aptos-labs/aptos-core#15542]
# 🐛 Bug

To report a security issue, please email security@aptoslabs.com.

## To reproduce

**Code snippet to reproduce**
```rust
# Your code goes here
# Please make sure it does not require any external dependencies
```

**Stack trace/error message**
```
// Paste the output here
```

## System information

**Please complete the following information:**
- Aptos Core Version
- Rust Version
- Computer OS

**Comment by gregnazario** [comment]
Fix here: https://github.com/aptos-labs/aptos-core/pull/15546

**Comment by gregnazario** [comment]
The fix has been released in 4.7.0
